.idea/codeStyles/Project.xml

```diff
@@ -19,6 +19,27 @@
     <JSCodeStyleSettings version="0">
       <option name="INDENT_CHAINED_CALLS" value="false" />
     </JSCodeStyleSettings>
+    <JetCodeStyleSettings>
```

Comment:
Any reason you are including this Project.xml which affects star imports for certain packages?

Reply:
No, although I have been changing my import settings to try to eliminate wildcard imports because An called them out to avoid and IntelliJ keeps resetting my preferences on them.  I'll revert this because I didn't try to set something this elaborate up purposefully.

Reply:
Reverted to be the same as the develop version.